ingest-app/src/cmr/ingest/api/subscriptions.clj

```diff
@@ -180,6 +180,20 @@
          (acl/verify-subscription-management-permission
            context :update :provider-object provider-id))))))
 
+(defn check-valid-user
+  "Raise error if the user provided does not exist"
+  [context user-id]
+  (when (or (nil? user-id)
+                (= "" user-id))
+    (errors/throw-service-error
+     :bad-request
+     "INGEST FAILED - Please provide a SubscriberId or pass in a valid token."))
```

Comment:
Could we see about making the exception message more descriptive about why the ingest failed?